### ISSUE TITLE
about queue_runners

### ISSUE THREAD

**Issue by zhangzj818** [ry/tensorflow-resnet#32]
when the program is running to"tf.train.start_queue_runners(sess=sess)"
then error.

ERROR:tensorflow:Exception in QueueRunner: Cast string to int64 is not supported
	 [[Node: Cast_1 = Cast[DstT=DT_INT64, SrcT=DT_STRING, _device="/job:localhost/replica:0/task:0/cpu:0"](Cast_1/x)]]
 my label dtype: int64 and img dtype:float32
how to fix? @ry  
thx
